### PR TITLE
Split observations in `media-observations` and `event-observations`

### LINKED ISSUE #254: How can observations be divided/grouped?
We currently state:

> ... a certain taxon, count, life stage, sex, behaviour and/or individual.
> 
> Number of observed individuals (optionally of given life stage, sex and behaviour).

Are those the 5 axes (taxon, life stage, sex, behaviour, individual) on which observations can be divided?
- Is `behaviour` one of them? I guess yes?
- Would `classifiedBy` be one of them (i.e. same observation, different AI/people that classified it)? In my opinion no.

I think we should clarify that within an event, it should be assumed that `count` can be summed up when grouping.

## Yes

```
scientificName | sex    | count
Vulpes vulpes  | male   | 1
Vulpes vulpes  | female | 1
```

=

```
scientificName | count
Vulpes vulpes  | 2
```

## No

```
scientificName | sex    | by | count
Vulpes vulpes  | male   | AI | 1
Vulpes vulpes  | female | AI | 1
Vulpes vulpes  | female | me | 1 
```

!=

```
scientificName | count
Vulpes vulpes  | 3
```

### PR DESCRIPTION
**Pull request by peterdesmet**
This PR implements https://github.com/tdwg/camtrap-dp/issues/203#issuecomment-1318656273 and splits `observations` into `media-observations` and `event-observations`. This supports media file based classification, event based classification or both. Having two tables makes the structure more clear and understandable, e.g. we don't have to express for each definition how it should be interpreted for media vs event based. It also allows to tables to diverge (i.e. adding specific fields that only make sense for one).

schema

## Names

While we use lowerCamelCase elsewhere, the [resource names have to be lowercase](https://specs.frictionlessdata.io/data-resource/#name). I opted for dash (`-`) to separate the words for better readability. We already use dash (and not `_` or `.`) in the the schema names.

```
media-observations.csv
media-observations-table-schema.json
event-observations.csv
event-observations-table-schema.json
```

## Structure

Both observations tables are currently exactly the same, except for:
- `mediaID` vs `eventID` as foreign key
- Media observations indicate: "Media observations are not required to be mutually exclusive, i.e. multiple classifications of the same observed individual(s) are allowed."
- Event observations indicate:  "Event observations are required to be mutually exclusive, so that `count` can be summed." Fix #254

## Term updates

- `observations.countNew` has been removed. It always had a slightly confusing definition and group size can now be expressed better in `event-observations`
- `deployment.timestampIssues` now refers to media table only (was still incorrectly mentioning the observations table)
- `media.eventID` the order of the sentences in the definition has been updated
- `profile.project.classificationLevel` has been removed. It now allows both and can be derived from the resources

## Website

Anchors to terms are now camelCase (rather than lowercase), which allows for easier copy/pasting, i.e. `/camtrap-dp/data/#event-observations.cameraSetupType`

## Example

The example dataset has been updated. It already contained a `media-observations.csv`, but that now includes the latest changes.

### CODE REVIEW

**Review by damianooldoni — APPROVED** [review]
Nice work. Definitely approved 👏 

Very few comments which probably can be tackled out of this PR:

- I see that `timestampIssues` is boolean. Maybe in the future could be interesting to have a text field (with controlled vocabulary?) to indicate the type of issues. Maybe the boolean field could remain but renamed as `timestampHasIssues`? Again, something you could take as an input for a later improvement stage.
- Some fields (e.g. `behavior`, `tag`) have in `description` the sentence `""Formatted as a pipe (`|`) separated list for multiple values, ..."`. It seems a MUST, but there is no control (constraint in table schema descriptor) on it. If not a MUST, I suggest to improve the description.

**Comment by peterdesmet** [comment]
@damianooldoni thanks for reviewing. Regarding `tags`, we could add regex, but it won't necessarily catch values that are incorrectly separated. But the regex could forbid `,`, `;` etc. if we don't want those characters in descriptions. Won't implement this for now.

**Review by kbubnicki — COMMENTED** [review]
Great job @peterdesmet! Only minor comments from my side.

File: media-observations-table-schema.json [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+      "type": "string",
+      "constraints": {
+        "required": true,
+        "unique": true
+      },
+      "example": "obs1"
+    },
+    {
+      "name": "mediaID",
+      "description": "Identifier of the event that was classified. Foreign key to `media.mediaID`.",
```
[the remark this answers is not in the document]
Comment (peterdesmet) [reply media-observations-table-schema.json:18]: Fixed in 8a37b72

File: media-observations-table-schema.json [hunk 2]
```diff
@@ -0,0 +1,217 @@
+{
+  "title": "Media observations",
+  "description": "Table with observations that are classified at media file level. Associated with media files via `mediaID`. Observations can mark non-animal events (camera setup, human, blank) or one or more animal observations (`observationType` = `animal`) of a certain taxon, count, life stage, sex, behavior and/or individual. Media observations are not required to be mutually exclusive, i.e. multiple classifications of the same observed individual(s) are allowed.",
```
[the remark this answers is not in the document]
Comment (peterdesmet) [reply media-observations-table-schema.json:3]: Rephrased in 810ba6c to:

> Media observations are not required to be mutually exclusive, i.e. multiple classifications (e.g. human vs machine) of the same observed individual(s) are allowed.

File: media-observations-table-schema.json [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+      "constraints": {
+        "required": true,
+        "unique": true
+      },
+      "example": "obs1"
+    },
+    {
+      "name": "mediaID",
+      "description": "Identifier of the event that was classified. Foreign key to `media.mediaID`.",
+      "skos:broadMatch": "http://rs.tdwg.org/dwc/terms/eventID",
```
[the remark this answers is not in the document]
Comment (peterdesmet) [reply media-observations-table-schema.json:19]: Will ask @tucotuco

[the remark this answers is not in the document]
Comment (peterdesmet) [reply media-observations-table-schema.json:19]: Changed to `associatedMedia` in b59b4f9

**Pull request merged**